Modules/Numerics/Statistics/include/itkDistanceMetric.h

```diff
@@ -97,6 +97,10 @@ class ITK_TEMPLATE_EXPORT DistanceMetric : public FunctionBase<TVector, double>
   {
     // Test whether the vector type is resizable or not
     MeasurementVectorType m;
+    for (size_t i = 0; i < m_MeasurementVectorSize; ++i)
+    {
+      m[i] = 0;
+    }
```

Comment:
Thanks for asking, @jhlegarreta. Please consider list-initialization by an empty initializer list `{}`, instead of a `for` loop. So in this case, just: `MeasurementVectorType m{};`

Reply:
:+1: Thanks Niels. Done in db2418b.